examples/learning_graph_embedding_and_predicting.py

```diff
@@ -0,0 +1,114 @@
+"""Learning embedding of graph using Poincare Ball Model."""
+
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+
+
+import geomstats.backend as gs
+import geomstats.visualization as visualization
+from geomstats.datasets.prepare_graph_data import HyperbolicEmbedding
+from geomstats.datasets.utils import load_karate_graph
+from geomstats.learning.kmeans import RiemannianKMeans
+
+
+def main():
+    """Learning Poincaré graph embedding.
+
+    Learns Poincaré Ball embedding by using Riemannian
+    gradient descent algorithm. Then K-means is applied
+    to learn labels of each data sample.
+    """
+    gs.random.seed(1234)
+
+    karate_graph = load_karate_graph()
+
+    hyperbolic_embedding = HyperbolicEmbedding(karate_graph)
```

Comment:
It would be better with a syntax that follows scikit-learn's logic.
```
hyperbolic_embedding = HyperbolicEmbedding()
embeddings = hyperbolic_embedding.embed(karate_graph)
```
the object hyperbolic_embedding should not depend on the dataset that will be passed to it, right? It could theoretically be used to embed another dataset later?